extensions/default/src/MergeDataSource/utils.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Executes a function given an object and a path to the function.
+ * @param {Object} obj - The object containing the function to be executed.
+ * @param {string} path - The path to the function, separated by dots.
+ * @param {Array} args - An array of arguments to be passed to the function.
+ * @returns {*} - The result of the executed function.
+ */
+export function executeFunction(obj, path, args) {
```

Comment:
Consider if you can use the lodash get method for this.

Reply:
Updated, thanks!